src/components/ContextControlsItem.vue

```diff
@@ -1,7 +1,14 @@
 <script setup lang="ts">
 import { showBuffer } from '@src/infrastructure/prun-ui/buffers';
 
-defineProps<{ cmd: string; label?: string }>();
+const props = defineProps<{ cmd: string; label?: string }>();
+
+const commandAndArg = computed(() => {
+  const words = props.cmd.split(' ');
+  return words[0] === 'XIT'
+    ? [words[0] + ' ' + words[1], words.slice(2).join(' ') ?? '']
+    : [words[0], words.slice(1).join(' ') ?? ''];
+});
```

Comment:
Let's simplify a bit

```suggestion
const commandAndArg = computed(() => {
  const words = props.cmd.split(' ');
  let command = words.shift();
  if (command === 'XIT') {
    command += ' ' + words.shift();
  }
  return [command, words.join(' ')];
});
```

Also:
1. Rename it to `commandParts` - sounds a bit more natural
2. You don't need the `??` operator here since `words.join(' ')` is never `null` or `undefined`. Read https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing